test/powershell/Language/Scripting/Debugging/Debugging.Tests.ps1

```diff
@@ -0,0 +1,96 @@
+$script1 = @'
+'aaa'.ToString() > $null
+'aa' > $null
+"a" 2> $null | % { $_ }
+'bb' > $null
+'bb'.ToSTring() > $null
+'bbb'
+'@
+$script2 = @'
+"line 1"
+"line 2"
+"line 3"
+'@
+
+$root = git rev-parse --show-toplevel
```

Comment:
Do we really want to assume tests are under a repo and that `git` is installed?  Consider moving these tests back into Windows.
I think it'd be smarter to use $PSScriptRoot and relative paths.